### ISSUE TITLE
`executeExpression` sometimes gives `Compiler Internal Error: ID 3 defined in multiple scopes.`

### ISSUE THREAD

**Issue by farmaazon** [enso-org/enso#8431]
To reproduce: 

1. check out `wip/farmaazon/cb-preview` branch,
2. run new GUI with opened dev-tools 
3. Open project with the following Main.enso:
```
from Standard.Base import all
from Standard.Table import all
from Standard.Database import all
from Standard.AWS import all
import Standard.Visualization.Preprocessor

main =
    operator1 = "Press ENTER key to create a new node"    
    operator13289 = operator1.length 
```
4. Open Component Browser

At this point, GUI always receives a message: 
```
{
    "jsonrpc": "2.0",
    "method": "executionContext/visualizationEvaluationFailed",
    "params": {
        "contextId": "ba7d1bfa-7475-45e5-b77d-60c846fe821e",
        "visualizationId": "e821316a-794d-4519-b0ff-f0144c3cb8f0",
        "expressionId": "d5cf4c57-9e05-4148-a0a6-301cca5ef277",
        "message": "Compiler Internal Error: ID 3 defined in multiple scopes.",
        "diagnostic": null
    }
}
```
And no value is send by binary connection.

**Comment by somebody1234** [comment]
potentially related to this issue?
- #8429

**Comment by 4e6** [comment]
Related to inline execution, but those are different issues

**Comment by 4e6** [comment]
After the investigations, it looks to me that this can be caused by #8428

**Comment by 4e6** [comment]
fixed by #8428 

:warning: Right now for the `Standard.Visualization.Preprocessor.default_preprocessor` expression to work, the `import Standard.Visualization` import is still needed in the module. It will be addressed by a followup issue.
